src/core/scene/a-scene.js

```diff
@@ -1,7 +1,6 @@
 /* global Promise, screen, CustomEvent */
 var initMetaTags = require('./metaTags').inject;
 var initWakelock = require('./wakelock');
-var loadingScreen = require('./loadingScreen');
```

Comment:
We have loading screen disabled, so none of these code paths were ever hit. Removed them instead.